package.json

```diff
@@ -50,7 +50,7 @@
   },
   "dependencies": {
     "@polkadot/api": "^5.8.4-1",
-    "@polkadot/apps-config": "^0.95.1",
+    "@polkadot/apps-config": "^0.95.2-104",
```

Comment:
Should this be listed in the resolutions as well with the other polkadot deps?

Reply:
Looks like `0.95.2-106` is out too. Worth bumping here?